jmdaemon/jmdaemon/onionmc.py

```diff
@@ -1397,7 +1445,7 @@ def wait_for_directories(self) -> None:
     """ CONTROL MESSAGES SENT BY US
     """
     def send_peers(self, requesting_peer: OnionPeer,
-                   peer_filter: List[OnionPeer]) -> None:
+                   peer_filter: List[OnionPeer], d: bool=False) -> None:

```

Comment:
We could use a more meaningful name instead of `d` (`has_left`, `have_left`, `disconnected`, etc.). A one-liner in the docstring about what that is could also help, though a better name could be self-explanatory already.

Reply:
Agreed, will do.

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/1249/commits/706bdb0a0379ed6ed841ee02c7fc77dff0c22b7b ; also re-wrote the docstring of `send_peers` a bit.